docs/vm-images.md

```diff
@@ -1,35 +1,33 @@
 # Supported and Tested VM Images
 
-* [HPC CentOS 7 VM Image](#hpc-centos-7-vm-image)
-* [Ubuntu](#ubuntu)
+* [HPC CentOS 7](#hpc-centos-7)
+* [HPC Rocky Linux 8](#hpc-rocky-linux-8)
+* [Debian 11](#debian-11)
+* [Ubuntu 20.04 LTS](#ubuntu-2004-lts)
 * [Other Images](#other-images)
 
-## HPC CentOS 7 VM Image
+## HPC CentOS 7
+
 The HPC Toolkit has officially supported the [HPC CentOS 7 VM Image][hpcimage] as the
 primary VM image for HPC workloads on Google Cloud since it's release. Since the
 [HPC CentOS 7 VM Image][hpcimage] comes pre-tuned for optimal performance on
 typical HPC workloads, it is the default VM image in our modules, unless there
 is specific requirement for a different OS distribution.
 
-Exceptions:
+[hpcimage]: https://cloud.google.com/blog/topics/hpc/introducing-hpc-vm-images
 
-* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
-  supported images [documented][exascalerimages] in the exascaler-cloud-terraform
-  github repo. **This requirement only applies to the servers, not the clients.**
-* [omnia-install]: Only provides support for Rocky 8.
+## HPC Rocky Linux 8
```

Comment:
We give a lot more info/help on Ubuntu than we do for Rocky or Debian. I feel they should all read about the same in the end. Maybe we could just point to https://cloud.google.com/hpc-toolkit/docs/setup/hpc-blueprint-catalog for users to find examples.

Reply:
I've added links to both the [HPC Blueprint Catalog](https://cloud.google.com/hpc-toolkit/docs/setup/hpc-blueprint-catalog) and [SchedMD's list of Slurm images](https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#public-image) at the top of the page..